### DISCUSSION TITLE
Copilot metrics

### DISCUSSION THREAD

**Discussion by wsantos** [community/community#58687] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

I'm writing to inquire about the possibility of accessing GitHub Copilot's metrics. I am particularly interested in understanding:

The rate at which Copilot's suggestions are accepted by users.
The overall usage statistics for Copilot.
Having access to this data will help us make more informed decisions on the tool's usefulness for our development process and potentially guide us in providing useful feedback for its continuous improvement.

Is there a way we can obtain these metrics or an interface through which we can analyze these data?

**Comment by davecheney** [comment, marked as answer]
@wsantos please reach out to your GitHub sales lead who can assist you.

**Comment by wsantos** [comment]
@davecheney can you tell me how exactly I should do it? because I've opened a support ticket and they asked me to open a message here and now you are redirecting again, how do I contact sales in github?

**Reply by davecheney** [reply]
I'm sorry for the runaround, i've shared this thread with the Copilot sellers. If your company has a GitHub Teams or enterprise subscription you may have a seller who is in contact with your procurement department.

**Reply by wsantos** [reply]
Got it, thank you.

**Comment by ninja-asa** [comment]
Hello, 
I have a Github Co-Pilot Licence for Individuals License. How should I proceed to collect those metrics, as there is not sales lead assigned?

Thank you in advance!

Sofia

**Reply by davecheney** [reply]
Please contact support, they can assist you or escalate as appropriate.
